### DISCUSSION TITLE
合作与秩序 ——「开源之法」小组的构建想法与实践 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#74] [Announcements]
# 合作与秩序 ——「开源之法」小组的构建想法与实践 - 「开源之道」

https://opensourceway.community/posts/os-license-and-copyleft/build-os-licensing-workgroup/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

报名:

大妈, CPyUG 联合创始人, 野生技术作者, 以及开源教育探索者;

**Comment by zhuangbiaowei** [comment]
报名，求拉群
